contents/handbook/growth/sales/sales-operations.md

```diff
@@ -75,20 +72,16 @@ _How do I find out a customer's usage?_
 
 _Can a customer transfer from self-hosted (e.g. Open Source) to Cloud?_
 
-Unfortunately we don't have a way to do this easily right now. If they have been on a Scale Paid plan, we can do this manually. If they are coming from the Open Source version, we suggest that they just restart on Cloud. 
+Unfortunately we don't have a way to do this easily right now. If they have been on a Scale plan, we can do this manually. If they are coming from the Open Source version, we suggest that they just restart on Cloud. 
 
-_Can a customer transfer from Cloud to Scale Paid?_
+_Can a customer transfer from Cloud to Scale?_
 
-Yes - we offer 3 months free of Scale Paid so they can run both systems in parallel, tracking events in both places. That means when they switch off Cloud, they'll have 3 months of data to start with in Scale.
+Yes - we offer 3 months free of Scale so they can run both systems in parallel, tracking events in both places. That means when they switch off Cloud, they'll have 3 months of data to start with in Scale.
```

Comment:
This answer has changed:

```
We offer plugins for customers to export all their data to a data warehouse. Then they can [bulk import historical data](https://posthog.com/docs/integrate/ingest-historic-data).

They will _lose_ their dashboard and configuration - this will just retain their events, but usually that's the principal concern.

If they need a more complex migration with many pre-made dashboards and lots of users and actions, we are currently trialling a partnership with another company to do this for them for a one off fee. Speak to James H if you have a client interested in this.
```

Reply:
Thanks - now updated